portality/bll/services/application.py

```diff
@@ -251,6 +251,22 @@ def accept_application(self, application, account, manual_update=True, provenanc
 
         return j
 
+    def reject_update_request_of_journal(self, journal_id, account):
+        """
+            Rejects update request associated with journal
+
+            :param journal_id:
+            :param account:
+            :return: Journal object
+        """
+        ur = models.Suggestion.find_latest_by_current_journal(journal_id)  # ~~->Application:Model~~
```

Comment:
`Suggestion` is the old model name, kept around for back-compat of old code, we use `models.Application` in new code